### PR TITLE
refactor: remove SinkRow

### PR DESCRIPTION
**Pull request by laysakura**
## Issue number and link

Refs: #189 

(de)serializing codes should appear only in Row, not spread in {Sink,Source}Row.

## Describe your changes

- Remove SinkRow and use Row instead.
- Move unit tests from SinkRow into Row

## Checklist before requesting a review

- [x] I follow the [Semantic Pull Requests](https://github.com/zeke/semantic-pull-requests) rules (bugfix/feature)
- [x] I specified links to related issues (must: bugfix, want: feature)
- [x] I have performed a self-review of my code (bugfix/feature)
- [ ] I have added thorough tests (bugfix/feature)
- [ ] I have edited `## [Unreleased]` section in `CHANGELOG.md` following [keep a changelog](https://keepachangelog.com/en/1.0.0/) syntax (bugfix/feature)
- [ ] I {made/will make} a related pull request for [documentation repo](https://github.com/SpringQL/SpringQL.github.io) (feature)

### CODE REVIEW

File: springql-core/src/api.rs [hunk 1]
```diff
@@ -13,12 +13,9 @@ pub use crate::{
         SpringValue,
     },
 };
-
-// This file is part of https://github.com/SpringQL/SpringQL which is licensed under MIT OR Apache-2.0. See file LICENSE-MIT or LICENSE-APACHE for full license details.
-
```
Comment (laysakura) [inline springql-core/src/api.rs:18]: redundant comment

File: springql-core/src/connection.rs [hunk 2]
```diff
@@ -26,13 +26,13 @@ fn setup_logger() {
 /// 1 stream pipeline has only 1 connection.
 /// In other words, the lifecycle of Connection and internal stream pipeline are the same.
 #[derive(Debug)]
-pub struct Connection {
+pub(crate) struct Connection {
```
Comment (kazuk) [inline springql-core/src/connection.rs:29]: may be dont need `(crate)`

Comment (laysakura) [reply springql-core/src/connection.rs:29]: @kazuk Correct. But If I change it into `pub`, I need to make many other structs `pub` due to `can't leak crate-private type` errors.

May I ask you to fix the visibility in https://github.com/SpringQL/SpringQL/pull/178 ?

Comment (kazuk) [reply springql-core/src/connection.rs:29]: Oh, yes. field types are defined as `pub(crate)` . Connection can not be `pub`
I will fix in #178

**Pull request merged**